plugin/src/com/microsoft/alm/plugin/external/commands/Command.java

```diff
@@ -229,6 +229,9 @@ public void completed(T result, Throwable error) {
         try {
             Throwable error = syncError.get();
             if (error != null) {
+                if (error.getMessage().contains("tf eula")) {
```

Comment:
Is this reliable? Is there a chance another string could contain `"tf eula"`. It doesn't seem like a common sequence of characters, but it's worth the ask - text matching always makes me nervous.

Reply:
I don't know any other way to handle EULA problem. As I understand the previous solution based on text "tf eula" in IDEA error message.